docs/cli/cli-data.md

```diff
@@ -158,7 +158,6 @@ And you can also just get the ID, using `jq`
 planet data search SkySatCollect --sort 'acquired desc' --limit 1 - | jq -r .id
 ```
 
-
```

Comment:
On line 149, suggest changing:

The lets you do things like get the id of the most recent skysat image taken (that you have download access to):

To:

This lets you do things like get the ID of the most recent SkySat image taken (and that you have permissions to download):